pkg/adapter/args.go

```diff
@@ -0,0 +1,27 @@
+package adapter
+
+import (
+	"flag"
+)
+
+type Args struct {
+	// Key defines the filename for tls Key.
+	Key string
+	// Cert defines the filename for tls Cert.
+	Cert string
+}
+
+var (
+	tlsCertFlag = flag.String("tls-cert", "",
+		"The filename for the TLS certificate.")
+	tlsKeyFlag = flag.String("tls-key", "",
+		"The filename for the TLS key.")
+)
+
+func GetArgs() Args {
```

Comment:
hopefully there is no more Args or we will have to start to use cobra because the builtin flags library is cumbersome to use.